src/App.js

```diff
@@ -18,14 +18,15 @@ import {
 library.add(faDotCircle, faMinusCircle, faPen, faSave, faTrash);
 
 class App extends Component {
+  componentDidMount() {}
   render() {
     return (
       <DragDropContextProvider backend={HTML5Backend}>
         <div className="container">
-          <h1>Test Structural Metadata Editor</h1>
-          <WaveformContainer />
+          <br />
+          <WaveformContainer {...this.props.config} />
```

Comment:
Here, the `this.props.config` seems odd.  I'd say let's extend the pattern of letting props flow through the components instead of adding on the extra `config` prop name.  So here and the same instance below, can we change to `<WaveformContainer {...this.props} />?   

And then in `Root.js`, replace `<App config={props} />` with `<App {...props} />`